README.md

```diff
@@ -11,19 +11,36 @@ and attacher for Ceph RBD and CephFS.
 
 ## Overview
 
-Ceph CSI plugins implement an interface between CSI enabled Container
-Orchestrator (CO) and CEPH cluster.
-It allows dynamically provisioning CEPH volumes and attaching them to
-workloads.
-Current implementation of Ceph CSI plugins was tested in Kubernetes
-environment (requires Kubernetes 1.13+), but the code does not rely on
-any Kubernetes specific calls (WIP to make it k8s agnostic) and
-should be able to run with any CSI enabled CO.
+Ceph CSI plugins implement an interface between CSI enabled Container Orchestrator
+(CO) and CEPH cluster. It allows dynamically provisioning CEPH volumes and
+attaching them to workloads.
 
-For details about configuration and deployment of RBD and
-CephFS CSI plugins, see documentation in `docs/`.
+Independent CSI plugins are provided to support RBD and CephFS backed volumes,
 
-For example usage of RBD and CephFS CSI plugins, see examples in `examples/`.
+- For details about configuration and deployment of RBD and CephFS CSI plugins,
+  see documentation in `docs/`.
+- For example usage of RBD and CephFS CSI plugins, see examples in `examples/`.
+
+## Project status
+
+Status: **Alpha**
+
+The **alpha** status reflects possible non-backward compatible changes in the
+future, and is thus not recommended for production use. There is work in progress
+that would change on-disk metadata for certain operations, possibly breaking
+backward compatibility.
```

Comment:
@ShyamsundarR This  ( alpha mention) gives a negative warning to the USER, I dont think we have to pass that message. We havent made any releases yet, so its pretty clear that there is NO stable version or we claim its stable. Lets handle this part by release process.

Reply:
I think 0.3 and 1.0 are considered releases, there is also some confusion in this regard with users. If there are clarifying statements provided to the contrary this can be changed. BUT, I do not see any -ve connotation here, just a call out of status. If not for Alpha what is the status you will call out?

Reply:
>I think 0.3 and 1.0 are considered releases, there is also some confusion in this regard with users
>
Not sure why we are concluding that. No releases available only development branches available. 

>BUT, I do not see any -ve connotation here, just a call out of status. If not for Alpha what is the status you will call out?
>

My point is that we could get rid of this section (alpha) completely and according to the release readiness or stability or naming we can update the readme or release doc

Reply:
So we shouls call out the status as "under development" then? or something to state this is not released? I ask as omiting it does not give anyone an idea about what the status of the plugin is, and that needs to be called out somewhere.

Reply:
Hmmm.. as this is an interim thing, I think its fine.. Lets keep it there as it is. Any way we will have a note with the release and can update the README accordingly.